pkg/controller/configmap_ca_injector/controller.go

```diff
@@ -124,7 +124,7 @@ func (r *ReconcileConfigMapInjector) Reconcile(request reconcile.Request) (recon
 
 		configMapList := &corev1.ConfigMapList{}
 		selector := labels.Set(map[string]string{names.TRUSTED_CA_BUNDLE_CONFIGMAP_LABEL: "true"}).AsSelector()
-		err = r.client.List(context.TODO(), &client.ListOptions{LabelSelector: selector}, configMapList)
+		err = r.client.List(context.TODO(), configMapList, &client.ListOptions{LabelSelector: selector})
```

Comment:
what!?!? did they seriously change the Client interface? Or was this a bug?

Reply:
unsure when, but yeah, interface has the target obj first, filter second. From what I can see it's been like that a couple releases